db/queries/application/queries.py

```diff
@@ -178,6 +184,24 @@ def get_count_by_status(round_ids: Optional[list] = [], fund_ids: Optional[list]
     return results
 
 
+def create_qa_base64file(application_data: dict, **params):
```

Comment:
Similarly here - we only need `with_questions_file` as a parameter instead of all `params`.

Reply:
Fixed